public/javascripts/big.js

```diff
@@ -620,25 +620,23 @@ lichess.storage = {
 
     function userPowertips() {
       var header = document.getElementById('site_header');
-      $('.ulpt').removeClass('ulpt').each(function() {
-        $(this).powerTip({
-          fadeInTime: 100,
-          fadeOutTime: 100,
-          placement: $(this).data('placement') || ($.contains(header, this) ? 'e' : 'w'),
-          mouseOnToPopup: true,
-          closeDelay: 200
-        }).on({
-          powerTipPreRender: function() {
-            $.ajax({
-              url: ($(this).attr('href') || $(this).data('href')).replace(/\?.+$/, '') + '/mini',
-              success: function(html) {
-                $('#powerTip').html(html);
-                $('body').trigger('lichess.content_loaded');
-              }
-            });
-          }
-        }).data('powertip', ' ');
-      });
+      $('.ulpt').removeClass('ulpt').powerTip({
+        fadeInTime: 100,
+        fadeOutTime: 100,
+        placement: $(this).data('placement') || ($.contains(header, this) ? 'e' : 'w'),
```

Comment:
Tested in chromium and surf and works with multiple usernames on page, it still refers to the same link tag as before the change in both browsers. Also tested on windows in chrome and firefox 34, seemed to work. Tried in internet explorer 8 but nothing worked there so don't know...